rotkehlchen/db/upgrades/v32_v33.py

```diff
@@ -0,0 +1,23 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from sqlite3 import Cursor
+
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def _update_fee_for_existing_trades(cursor: 'Cursor') -> None:
+    cursor.execute("""
+    UPDATE trades
+    SET fee = NULL
+    WHERE fee_currency IS NULL
+    """)
+
+
+def upgrade_v32_to_v33(db: 'DBHandler') -> None:
```

Comment:
The upgrade should be in the existing v31 to v32. No need to make a new one. A new DB upgrade is made only at release time.